<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>다음_Regular</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ForStudent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toGradeCalculator">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>등급계산기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toSplitGrade">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>등수 to 등급</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
